content/articles/delegating-dnsimple-registered.md

```diff
@@ -23,18 +23,22 @@ Switching the name servers to DNSimple will cause the domain to resolve using th
 
     ![Domain Delegation card](/files/domain-delegation-card-2.png)
 
-1.  Enter the [DNSimple name servers](/articles/dnsimple-nameservers)
+1. Click on <label>Use DNSimple name servers</label> to replace the name servers in the form with [DNSimple name servers](/articles/dnsimple-nameservers). 
+
+    ![Use DNSimple name servers](/files/use-dnsimple-name-servers.png)
+
+1. Click on <label>Use DNSimple name servers</label> in the following confirmation dialog.
+
+    ![Use DNSimple name servers confirmation](/files/use-dnsimple-name-servers-confirmation.png)
+
+1. Alternatively, the [DNSimple name servers](/articles/dnsimple-nameservers) can also be keyed into the form manually
```

Comment:
```suggestion
1. Alternatively, the [DNSimple name servers](/articles/dnsimple-nameservers) can also be typed into the form manually
```

Reply:
Don't trust the 🇪🇸 for this, but I think _typed_ or _entered_ makes more sense in this context.

Reply:
Thanks for the feedback @jacegu @itsalyse ! I have made the amendment here: https://github.com/dnsimple/dnsimple-support/pull/1287/commits/07d67eb2c257e8166d371d62be591b00fe17456a